src/HomeScreen.tsx

```diff
@@ -1,29 +1,54 @@
-import React, { Component } from "react";
+import React, { Component } from 'react';
 
-import { VoteButton } from "shared";
+import { VoteButton } from 'shared';
 
 // TODO: smh think of a better name lol
-const BoxInput = ({title, iconClassname}: {title: string, iconClassname: string}) => (
-  <button className="btn home-screen__box-input" onClick={() => alert(title)}>
-      <div className={iconClassname}/>
-      <div className="type--large-white">{title}<span className="type--large-green">.</span></div> 
-    </button>
+const BoxInput = ({
```

Comment:
What do you think about having BoxInput in its own file? So like maybe: 
feature/
    Home/
          Containers(Or screens idk)/
          Components/
                BoxInput.tsx

Reply:
uhh If it's used only within homeScreen I'd prefer to leave it in here for two reasons: 

1. it's a small enough component
2. it's tightly coupled with the home screen component

If we're going to be using it elsewhere though then it voids my second reason and I think it makes sense to put it in it's own file. Are you strongly for putting it in it's own file?